presto-cache/src/main/java/com/facebook/presto/cache/alluxio/AlluxioCachingFileSystem.java

```diff
@@ -63,6 +68,7 @@ public synchronized void initialize(URI uri, Configuration configuration)
                 throw new IOException("Failed to open file", e);
             }
         });
+        this.cacheQuotaEnabled = configuration.getBoolean(PropertyKey.USER_CLIENT_CACHE_QUOTA_ENABLED.getName(), false);
```

Comment:
nit: can we static import `USER_CLIENT_CACHE_QUOTA_ENABLED`?

Reply:
done